stdlib/public/TensorFlow/TensorHandle.swift

```diff
@@ -16,33 +16,49 @@
 
 import CTensorFlow
 
-/// `_AnyTensorHandle` is the scalar-agnostic base type for `TensorHandle`, used
-/// specifically for low-level, type-erased passings of Swift-level tensor
-/// handles in the compiler.
-@_fixed_layout // required because the compiler accesses _cTensorHandle directly.
-public class _AnyTensorHandle {
+/// This protocol abstracts the underlying representation of a tensor. Any type
+/// that conforms to this protocol can be used as a `TensorHandle` in the
+/// `TensorFlow` library, as it much provide a way to convert the underlying tensor
+/// handle into a `ConcreteTensorHandle`, which wraps a `TFE_TensorHandle *`
+public protocol _AnyTensorHandle {
+  var tfeTensorHandle: TFETensorHandle { get }
```

Comment:
All properties and methods in underscored protocols should start with an underscore, so that `tfeTensorHandle` won't be leaked through code completion.

```suggestion
  var _tfeTensorHandle: TFETensorHandle { get }
```

Reply:
Done.